react/components/MyUsers.tsx

```diff
@@ -44,7 +45,8 @@ const MyUsers = ({
   intl,
 }: Props) => {
   const PAGE_SIZE_STEPPER = 100
-
+  const [searchUser, setSearchUser] = useState('')
+  const [assignmentsToRender, setAssignmentsToRender] = useState<OrganizationAssignment[]>([])
```

Comment:
this doesn't need to be a state as it doesn't have any "interaction" in the page. We can store the results in a variable, and simplify the lifecycle.

Reply:
Changed!